cmd/client/main.go

```diff
@@ -52,7 +52,7 @@ var (
 	tokenPath          = kingpin.Flag("token-path", "Uses an OAuth 2.0 Bearer token found in this path to make scrape requests").String()
 	insecureSkipVerify = kingpin.Flag("insecure-skip-verify", "Disable SSL security checks for client").Default("false").Bool()
 	useLocalhost       = kingpin.Flag("use-localhost", "Use 127.0.0.1 to scrape metrics instead of FQDN").Default("false").Bool()
-	allowPort          = kingpin.Flag("allow-port", "Restricts the proxy to only being allowed to scrape the given port").Default("*").String()
+	allowPorts         = kingpin.Flag("allow-port", "Restricts the proxy to only being allowed to scrape the given ports, separated by commas").Default("*").String()
```

Comment:
Based on joshmeranda's previous feedback:

```suggestion
	allowPorts         = kingpin.Flag("allow-port", "Restricts the proxy to only being allowed to scrape the given ports, separated by commas").Default("*").Strings()
```